spec/spec_helper.rb

```diff
@@ -3,26 +3,4 @@
 
 require 'voxpupuli/test/spec_helper'
 
-def get_content(subject, title)
```

Comment:
These are modulesynced. So while I appreciate the removal, we're not really ready for it just yet :)

Reply:
Done.